test/server/specification.test.js

```diff
@@ -151,7 +151,7 @@ describe('Specification', () => {
 
       ['#4', 'Fix #4', 'user/repo#42', 'closes user/repo#42',
         'some-org/repo#42', 'http://tracker.com', 'https://tracker.com',
-        'www.issues.example.com', 'Fix for http://some.tracker.com/issues/42'
```

Comment:
Yes. The previous regex matched URLs without the protocol (or with bogus protocols like smb://), and I'm not sure if it's really necessary. I just took the simplest possible one, which might be more lax but on the other hand probably won't freeze at bogus inputs.